mxcubecore/HardwareObjects/abstract/AbstractAperture.py

```diff
@@ -53,21 +53,24 @@ def init(self):
             logging.getLogger("HWR").error("Aperture: no position list defined")
 
     def get_diameter_size_list(self):
-        """
+        """Get diameters size list
+
         Returns:
             list: list of diameter sizes in microns
         """
         return self._diameter_size_list
 
     def get_position_list(self):
-        """
+        """Get positions list
```

Comment:
```suggestion
        """Get list of positions.
```

and so on ...

It's not always needed; for instance, the next one reads fine: "Get current diameter index"